src/pages/Overview/index.tsx

```diff
@@ -1,7 +1,18 @@
-import React from 'react';
+import { ActionContainer } from './styled';
+import InfoCards from '../../components/overview/InfoCards';
+import TasksInfoBox from '../../components/overview/tasksBox';
+import UnresolvedInfoBox from '../../components/overview/unresolvedInfoBox';
 
 const Overview = () => {
-  return <div>Overview</div>;
+  return (
+    <>
+      <InfoCards />
+      <ActionContainer>
+        <UnresolvedInfoBox />
+        <TasksInfoBox />
```

Comment:
There are always different name of folder and component. It makes to hard to navigate through the app

Reply:
fix: tasksBox => tasksInfoBox